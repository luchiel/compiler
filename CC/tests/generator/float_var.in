double d;

void main()
{
double loc;
d = 4.5;
loc = 5.5;
printf("local: %.1f, global: %.1f\n", loc, d);
loc = d;
printf("local from global: %.1f\n", loc);
d = loc;
printf("global from local: %.1f", d);
}