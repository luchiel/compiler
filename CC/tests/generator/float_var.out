local: 5.5, global: 4.5
local from global: 4.5
global from local: 4.5